autobuild/app.py

```diff
@@ -22,7 +22,7 @@
 parser.add_argument('--testnet', help='Use testnet', default=False)
 parser.add_argument('--syncmode', help='sync mode', default='light')
 parser.add_argument('--gethexternal', help='Use remote ethereum node', default=False)
-parser.add_argument('--custom_manifest', help='URL to custom manifest files and configs', default=False)
+parser.add_argument('--custom_manifest', help='raw URL to custom manifest file and configs; e.g. https://raw.githubusercontent.com/ConanMishler/blockchain-configuration-files/bump-SYS-v4.2.2/', default=False)
```

Comment:
I have the original master as default in my version of this - 
```
parser.add_argument('--branchpath', help='Custom branch path for testing configs', default='https://raw.githubusercontent.com/blocknetdx/blockchain-configuration-files/master')
```
and always pass the path down to utils/autoconfig.py but there's more than one way to skin a cat...

Reply:
Yeah, I probably would've done it like that too. But it works fine the way Leo did it.